RELEASE_NOTES.md

```diff
@@ -16,9 +16,16 @@
 * Refactor DeliveryPeriod to take in a timedelta duration attribute instead of the DeliveryDuration Enum type
 * Public trades renamed from public_trade_lists to public trades and all _lists suffixes removed
 * Remove ORDER_EXECUTION_OPTION_NONE from OrderExecutionOption
+* Add unit tests for the client types and functions
+* Add error handling in the client for the gRPC errors that could be raised by the service
+
 
 ## Bug Fixes
 
 * Remove `frequenz-api-common` files now that dependency conflict is solved
 * Fix DeliveryArea from and to pb methods
 * Use HasFields method on protobuf messages
+* Add a missing optional keyword to the `UpdateGridpoolOrderRequest` protobuf definition
```

Comment:
Seems that this has been done in a different PR. Correct?

Reply:
It made the tests fail that's why it's here

Reply:
True my bad this was subsequently done by you in #61 actually so I can remove it here

Reply:
Oh great stuff :D